home-exercises-05-2022/src/io/codelex/arithmetic/practice/TwoNumbers.java

```diff
@@ -0,0 +1,15 @@
+package io.codelex.arithmetic.practice;
+
+public class TwoNumbers {
+    static void twoNumbersSumOrDifferenceIsFifteen(int x, int y) {
+        if ((x | y) == 15 | (x + y) == 15 | Math.abs(x-y) == 15) {
```

Comment:
Te nepareizi tiek lietots bitwise or operators | 
Šādi tas strādā tikai ja x ir mazāks par y. Es ieteiktu bitwise operatorus neizmantot, jo to pielietojumam jābūt tehniski pamatotam. Vienkārši x == 15 || y == 15 :)